test/sqlsmith/mzcompose.py

```diff
@@ -23,7 +23,7 @@
 from materialize.sqlsmith import known_errors
 
 TOTAL_MEMORY = 8
-NUM_SERVERS = 2
+NUM_SERVERS = 8
```

Comment:
Is this supposed to be here?

Reply:
Sorry, no. That was for my local test run. Fixed